pkg/kubelet/cm/topologymanager/topology_manager.go

```diff
@@ -91,7 +92,7 @@ type HintProvider interface {
 	// Allocate triggers resource allocation to occur on the HintProvider after
 	// all hints have been gathered and the aggregated Hint is available via a
 	// call to Store.GetAffinity().
-	Allocate(pod *v1.Pod, container *v1.Container) error
+	Allocate(ctx context.Context, pod *v1.Pod, container *v1.Container) error
```

Comment:
we may want to add a context also into `Get{Pod,}TopologyHints`. In general all the calls in the `HintProvider` interface should have a context

Reply:
done